.dockerignore

```diff
@@ -0,0 +1,20 @@
+/.git
+/public/build
+/public/bundles
+/var/log/*
+var/sessions/*
+/vendor
+.dockerignore
+.editorconfig
+.env.local
+.env.test
+.gitignore
+.php_cs.dist
+.travis.yml
+appveyor.yml
+docker-compose.override.yml.dist
+docker-compose.yml
+Dockerfile
+Makefile
+phpunit.xml.dist
+README.md
```

Comment:
I suggest add also:
* `/docker`
* `/node_modules` (might appear while development, where drive is mounted in docker-compose)

Reply:
Poissible to miss some files yes, I've just copy a file from my repository, but I run the build in Travis, then some folder/files are never present.

For the docker folder, I can't because the Dockerfile need to access it.

Reply:
Ah, yes, in case of CI/CD those folders probably wont appear. Tests outputs are also isolated in your makefiles. Just missed it checking build locally.